client/src/main/java/com/alibaba/nacos/client/security/SecurityProxy.java

```diff
@@ -115,6 +115,7 @@ public boolean login(List<String> servers) {
                 }
             }
         } catch (Throwable ignore) {
+            SECURITY_LOGGER.error("[SecurityProxy] login  failed, servers: {}, errorMsg: {}", servers,  ignore.getMessage());
```

Comment:
If the throwable is not ignored, maybe shoule rename.

Reply:
Did you mean that i should try to log user's password and rename the ignore to e or ex?

Reply:
1. I think we should not print password, If necessary, only print with `debug` level.
2. ignore means the exception don't need to care. if you log this exception, it should not named ignore, need be rename.